test/index.js

```diff
@@ -1338,3 +1338,22 @@ describe('parse', function() {
   });
 
 });
+
+describe('regex', function() {
+
+  describe('match', function() {
+
+    it('returns a Just if there is a match', function() {
+      eq(S.match(/(good)?bye/, 'goodbye').toString(), 'Just(["goodbye", "good"])');
+      eq(S.match(/([a-z]a)/g, 'bananas').toString(), 'Just(["ba", "na", "na"])');
```

Comment:
I lifted that banana one direct from the Ramda docs! Dunno why but I've always found it funny so wanted to add it in.